templates/zerver/help/configure-authentication-methods.md

```diff
@@ -0,0 +1,27 @@
+# Configure authentication methods
+
```

Comment:
Typically there's a brief sentence here introducing the scope of the article; in this case, you should explain what authentication methods are and why organizations might need different ones, etc.